bpmsuite/bpmsuite70-businesscentral-monitoring-with-smartrouter.json

```diff
@@ -144,14 +129,15 @@
             "description": "KIE server controller service (Used to set the org.kie.server.controller system property if host and port aren't set)",
             "name": "KIE_SERVER_CONTROLLER_HOST",
             "example": "my-app-controller.os.mycloud.com",
-            "required": false
+            "required": true
```

Comment:
@rbarriuso I think we don't have to set this variable as required. It can be derived from KIE_SERVER_CONTROLLER_SERVICE.

Reply:
Right. Fixed!